bitnami/kubeapps/Chart.yaml

```diff
@@ -30,4 +30,4 @@ maintainers:
 name: kubeapps
 sources:
   - https://github.com/vmware-tanzu/kubeapps
-version: 8.0.19
+version: 8.0.20
```

Comment:
I don't know if this change is seen as a patch version instead of a minor. Let's have someone from the team confirm it first.

Reply:
Same here. Was a worthwhile point to consider, but as the chart had been landed, I didn't come back to it, sorry. Luckily, the minor version was bumped with the next PR that landed here upstream anyway :)